packages/tame-metering/src/tame.js

```diff
@@ -76,7 +79,8 @@ export function tameMetering() {
     if (
       typeof target !== 'function' ||
       target === FunctionPrototype ||
-      target === SES1ErrorConstructor
+      target === SES1ErrorConstructor ||
+      target === globalEval
```

Comment:
Maybe clarify in the comment above why this “hack” does not breach containment (I’m not sure whether this is the case). Would be good to clarify too that SES-1.0 is actually SES before 0.8.0, and if/why it’s not necessary later. Sounds like it’s still necessary after 0.8.0.

Reply:
Not necessary after 0.8.0.  I've clarified.